arroyo-openapi/api-spec.json

```diff
@@ -8,6 +8,11 @@
     },
     "version": "1.0.0"
   },
+  "servers": [
```

Comment:
it looks like this is still out of sync -- I think we discussed gitignoring this?

Reply:
possibly related, even after rebuilding I get this error on the swagger UI:

![image](https://github.com/ArroyoSystems/arroyo/assets/100920/74590ab9-6c28-4afb-90ad-cbaa291752e0)

Reply:
Fixed